org.metaborg.meta.lang.template/trans/analysis/desugar.str

```diff
@@ -216,44 +216,44 @@ rules
   create-productions-align-list(|chars):
     SdfProduction(lhs, Rhs(rhs*), Attrs(attrs*)) -> prods*
     where
-      pos*           := <collect-all(?Align(<id>), collapse-lists); not(?[])> attrs*;
```

Comment:
I'm not sure this change is necessary / correct. I know the strategy name might be misleading, but isn't this being used to generate `align` productions, and not `align-list`? Did you change the constructors in your new syntax?